atomicapp/providers/kubernetes.py

```diff
@@ -203,4 +203,8 @@ def undeploy(self):
             cmd = [self.kubectl, "delete", "-f", path, "--namespace=%s" % self.namespace]
             if self.config_file:
                 cmd.append("--kubeconfig=%s" % self.config_file)
-            self._call(cmd)
+            try:
+                self._call(cmd)
```

Comment:
wouldn't it be best to add `ignore_error` to `_call()` and then `_call()` and call `run_cmd()` `checkexitcode=` arg set?

In that case the try/except would be all the way in `run_cmd` which already has this functionality built in.